temporal-sdk/src/main/java/io/temporal/workflow/ChildWorkflowOptions.java

```diff
@@ -315,6 +300,15 @@ public Builder setCronSchedule(CronSchedule c) {
       return this;
     }
 
+    /**
+     * Specifies whether this child workflow should run on a worker with a compatible build ID or
+     * not. See the variants of {@link VersioningIntent}.
+     */
+    public Builder setVersioningIntent(VersioningIntent versioningIntent) {
```

Comment:
Is there a reason normal workflow options do not need versioning intent? Maybe I am misunderstanding something.

Reply:
Because new workflows always start on the default. "Compatible" doesn't really make any sense for a new workflow, at least not without specifying a version, but we don't want to do that.